weave/flow/eval.py

```diff
@@ -441,7 +463,7 @@ async def eval_example(example: dict) -> dict:
             except Exception as e:
                 print("Predict and score failed")
                 traceback.print_exc()
-                return {"output": None, "scores": {}}
+                return {self._output_key: None, "scores": {}}
```

Comment:
this should always be "output" even if you have some scorers that use "model_output"